### PR TITLE
Display flags as rectangles

### PR DESCRIPTION
**Pull request by tarunsamanta2k20**
Please don't close the PR. Let me know if any changes are required.

Screenshot at 2023-03-05 09-58-45

### CODE REVIEW

**Comment by nikhilwoodruff** [comment]
Thanks @tarunsamanta2k20 for this (let me know if I got the intended issue fix wrong!). Let me know when it's ready for a review and I'll take a look!

**Comment by tarunsamanta2k20** [comment]
@nikhilwoodruff please review it now and let me know if any changes require.

**Comment by MaxGhenis** [comment]
Thanks @tarunsamanta2k20. In addition to @nikhilwoodruff's suggestion, could you please:
* Add new rectangle flags for the US and UK
* Edit your initial PR description to say `Fixes #267` - this will close that issue upon merging this PR

**Comment by MaxGhenis** [comment]
That's not the US flag. Please also name the flags as the two-letter country code, lowercase.

**Comment by tarunsamanta2k20** [comment]
> That's not the US flag. Please also name the flags as the two-letter country code, lowercase.

@MaxGhenis done

File: src/layout/CountrySelector.jsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { DownOutlined } from "@ant-design/icons";
 import { Dropdown } from "antd";
 import { useLocation, useNavigate } from "react-router-dom";
-import UKFlag from "../images/logos/countries/uk.webp";
-import USFlag from "../images/logos/countries/us.png";
+import UKFlag from "../images/logos/countries/uK.png";
+import USFlag from "../images/logos/countries/usa.png";
```
Comment (MaxGhenis) [inline src/layout/CountrySelector.jsx:5]:
```suggestion
import USFlag from "../images/logos/countries/us.png";
```

File: src/layout/CountrySelector.jsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import { DownOutlined } from "@ant-design/icons";
 import { Dropdown } from "antd";
 import { useLocation, useNavigate } from "react-router-dom";
-import UKFlag from "../images/logos/countries/uk.webp";
-import USFlag from "../images/logos/countries/us.png";
+import UKFlag from "../images/logos/countries/uK.png";
```
Comment (MaxGhenis) [inline src/layout/CountrySelector.jsx:4]:
```suggestion
import UKFlag from "../images/logos/countries/uk.png";
```

**Review by MaxGhenis — COMMENTED** [review]
please also remove USA.avif and drop a screenshot of it after making the changes

**Comment by MaxGhenis** [comment]
Thanks! Could you please mark resolved conversations as resolved?

@nikhilwoodruff I noticed we have some webp and png copies of the same flags in https://github.com/PolicyEngine/policyengine-app/tree/master/src/images/logos/countries. Should we remove those in this PR? Do we need both formats?

**Comment by nikhilwoodruff** [comment]
@MaxGhenis I don't think we do (removing them in this PR is good but not needed IMO)

**Review by nikhilwoodruff — APPROVED** [review]
Thanks @tarunsamanta2k20 - apologies for my delay in getting back to reviewing this. LGTM, merging!

**Comment by tarunsamanta2k20** [comment]
@nikhilwoodruff @MaxGhenis please add me as a member of your organization. I wanna contribute more in tte future.

**Comment by nikhilwoodruff** [comment]
Amazing, looking forward to it @tarunsamanta2k20 - could you introduce yourself on the #introductions channel in our [Slack](https://join.slack.com/t/policyengine-group/shared_invite/zt-1wnzujjgh-5JGi5O4z7Bi7Rh9kzIyZOA)?

**Pull request merged**